azurerm/automation_variable.go

```diff
@@ -0,0 +1,225 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/automation/mgmt/2015-10-31/automation"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func parseAzureRmAutomationVariableValue(resource string, input *string) (interface{}, error) {
```

Comment:
As this is only used one place should be just use it inline in the function?

Reply:
I actually like this as a separate helper function personally, I think the code is more readable that way...